preflight-checks/src/package.js

```diff
@@ -0,0 +1,14 @@
+const currentWorkingDirectory = process.cwd();
+const path = require( 'path' );
+const packageFile = path.join( currentWorkingDirectory, 'package.json' );
+const chalk = require( 'chalk' );
+
+let packageJson;
+try {
+	packageJson = require( packageFile );
+} catch( e ) {
+	console.error( chalk.red( 'Error:' ), `Could not find a ${ chalk.yellow( 'package.json' ) } in the current folder.` );
```

Comment:
Let's include the folder in case there are any issues:

```suggestion
	console.error( chalk.red( 'Error:' ), `Could not find a ${ chalk.yellow( 'package.json' ) } in the current folder (${ currentWorkingDirectory }).` );
```